W-17575001-salesforce-update-kt (#2770)
mulesoft/docs-connectors: salesforce/10.22/modules/ROOT/pages/salesforce-connector-reference.adoc

NOTE: Salesforce Connector supports providing a signed certificate only if you use the `SHA256withRSA` algorithm for the OAuth JWT connection type.

> Enabling "JSON Web Token-based access tokens" in Salesforce server is intended only for REST API calls and is incompatible with the Salesforce Connector

Reviewer: ```suggestion
You can only enable *JSON Web Token-based access tokens* option on Salesforce server for REST API calls. This option isn't compatible with Salesforce Connector.
```
